swift/apple/FirezoneKit/Sources/FirezoneKit/Models/Settings.swift

```diff
@@ -31,15 +31,15 @@ struct AdvancedSettings: Equatable {
         authBaseURLString: "https://app.firez.one/",
         apiURLString: "wss://api.firez.one/",
         connlibLogFilterString:
-          "connlib_client_apple=debug,firezone_tunnel=trace,phoenix_channel=debug,connlib_shared=debug,connlib_client_shared=debug,warn"
+          "connlib_client_apple=debug,firezone_tunnel=trace,phoenix_channel=debug,connlib_shared=debug,connlib_client_shared=debug,info"
       )
     #else
       AdvancedSettings(
         authBaseURLString: "https://app.firezone.dev/",
         apiURLString: "wss://api.firezone.dev/",
         connlibLogFilterString:
-          "connlib_client_apple=info,firezone_tunnel=trace,"
-          + "connlib_shared=info,phoenix_channel=info,connlib_client_shared=info,boringtun=debug,snownet=debug,str0m=info,firezone_tunnel=debug,warn"
+          "connlib_client_apple=info,firezone_tunnel=info,"
```

Comment:
Same here, it might be good to try to standardize on `debug` for development and `info` or `warn` for prod?

The debugging process with customers can be "set your log filter to `debug` and repeat the steps before exporting logs"

Reply:
> Same here, it might be good to try to standardize on `debug` for development and `info` or `warn` for prod?

Yeah I think that is a reasonable call. Instead of just doing pass-through of this variable, we could also make our own config and set the logger accordingly, i.e.: setting the logger to "debug" means we configure tracing with certain crates that we know are useful (like `boringtun`) and leave others out that we don't care about (like `rustls`).

Reply:
Hm, yeah, that might be a good idea. refs #3618 so we can come back to this idea.

Reply:
If we can do that in a way where the super-customizable log levels are still an option, that would be cool.

Maybe with a prefix or something, one prefix for the original syntax and another for these pet names